charts/hedera-json-rpc-relay-websocket/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Q: Is this resource needed?
The websocket server streams from the mirror node only, so the below SDK config secrets should not be needed